test/simps.lean

```diff
@@ -11,9 +11,27 @@ infix ` ≃ `:25 := equiv
 
 namespace foo
 
-@[simps] protected def rfl {α} : α ≃ α :=
+def field {α β : Sort*} : α ≃ β → pprod (α → β) (β → α)
+| ⟨f,g,_,_⟩ := ⟨f,g⟩
+
+@[projection equiv]
+def field_eq {α β : Type} (f : α → β) {g h h'} : field ⟨f,g,h,h'⟩ = ⟨f,g⟩ := rfl
```

Comment:
please add a test where the proof is not reflexivity. I think that test will currently fail.